ephios/plugins/basesignup/templates/basesignup/section_based/fragment_participant.html

```diff
@@ -1,20 +1,28 @@
 {% extends "core/disposition/base_participation.html" %}
+{% load user_extras %}
 {% load bootstrap4 %}
 {% load i18n %}
 {% block body %}
     <h6 class="mb-0">
         <div class="row align-items-center no-gutters">
-            <div class="col-6 pr-3">
-                <span class="fa fa-user"></span>
-                {{ form.instance.participant }}
-                {% if form.preferred_section %}
-                    <small class="text-secondary">
-                        <br> {% translate "prefers" %}
-                        {{ form.preferred_section.title }}
-                    </small>
-                {% endif %}
+            <div class="col-8 d-flex pr-3">
+                <div class="pr-1">
+                    <span class="fa fa-user"></span>
+                    {{ form.instance.participant }}
+                    {% if form.preferred_section %}
+                        <small class="text-secondary">
+                            <br> {% translate "prefers" %}
+                            {{ form.preferred_section.title }}
+                        </small>
+                    {% endif %}
+                </div>
+                <div>
+                    {% for qualification in form.instance.participant.qualifications|get_relevant_qualifications %}
+                        <span class="badge badge-secondary">{{ qualification }}</span>
+                    {% endfor %}
+                </div>
             </div>
-            <div class="col-6 d-flex justify-content-between align-items-center">
+            <div class="col-4 d-flex justify-content-between align-items-center">
                 {% bootstrap_field form.section show_label='sr-only' form_group_class="form-group d-inline-block mb-0" %}
 
                 {% if form.can_delete %}
```

Comment:
damn github I can't mark line 29 (new). Can we do `ml-1` instead of `ml-3` here? Space is a little wasted otherwise...

![image](https://user-images.githubusercontent.com/5970076/109708395-a5fe8400-7b9b-11eb-9c30-ee5ec1200fa3.png)

Reply:
this is not my code but yeah sure ^^

Reply:
Well you did make the columns smaller, didn't ya? :p